symplyphysics/laws/thermodynamics/work_done_by_gas_at_constant_pressure.py

```diff
@@ -0,0 +1,41 @@
+from sympy import (Eq, solve, integrate)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+# Common formula for work done by gas is A = ∫p * dV
+# A is work done
+# p is pressure
+# dV is change in volume
+
+
+pressure = Symbol("pressure", units.pressure)
+volume = Symbol("volume", units.volume)
+init_volume = Symbol("init_volume", units.volume)
+result_volume = Symbol("result_volume", units.volume)
+work = Symbol("work", units.energy)
+
+common_law = integrate(pressure, (volume, init_volume, result_volume))
```

Comment:
Why integrate it if it is constant?

Reply:
Because it is common formula for work of gase for any procces. I decided that I should derive this from the integral equation

Reply:
You either describe work done at constant pressure of use general formula, using integration.